usaspending_api/spending_explorer/v2/filters/explorer.py

```diff
@@ -96,18 +96,34 @@ def object_class(self):
 
     def recipient(self):
         # Recipients Queryset
+        # The two querysets here are identical, other than that the annotation traces differently to either the
```

Comment:
Are there performance issues when using a coalesce instead of creating two querysets?

I would assume for here no.

Although I do have questions for using coalesce in usaspending_api/spending_explorer/v2/filters/spending_filter.py